test/pools/PoolCollection.ts

```diff
@@ -375,20 +375,18 @@ describe('PoolCollection', () => {
                 poolTokenFactory.address,
                 poolMigrator.address
             );
-            expect(await poolCollection.version()).to.equal(11);
+            expect(await poolCollection.version()).to.equal(12);
 
             expect(await poolCollection.poolType()).to.equal(PoolType.Standard);
             expect(await poolCollection.defaultTradingFeePPM()).to.equal(DEFAULT_TRADING_FEE_PPM);
             expect(await poolCollection.networkFeePPM()).to.equal(DEFAULT_NETWORK_FEE_PPM);
             expect(await poolCollection.protectionEnabled()).to.equal(false);
 
             await expect(poolCollection.deployTransaction)
-                .to.emit(poolCollection, 'DefaultTradingFeePPMUpdated')
-                .withArgs(0, DEFAULT_TRADING_FEE_PPM);
+                .to.not.emit(poolCollection, 'DefaultTradingFeePPMUpdated');
```

Comment:
Unnecessary, IMHO

Reply:
Of course, I kinda left it as a "stub", to remind us that it should have been there.
Do you recommend removing it?